test/integration/good/compiler-optimizations/cpp.expected

```diff
@@ -1041,13 +1051,8 @@ class optimizations_model : public model_base_crtp<optimizations_model> {
         lp_accum__.add(159);
         double paramvar;
         
-        {
-          {
-            sym112__ = (theta * 34);
-          }
-        }
-        current_statement__ = 85;
-        lp_accum__.add(sym112__);
+        current_statement__ = 83;
+        lp_accum__.add(DUMMY_VAR__);
```

Comment:
This looks like an optimizer bug. Unoptimized code is
```cpp
        local_scalar_t__ paramvar;
        
        current_statement__ = 131;
        paramvar = DUMMY_VAR__;
        current_statement__ = 131;
        paramvar = 3;
        current_statement__ = 133;
        if (logical_gt(42, 1)) {
          current_statement__ = 132;
          paramvar = (theta * 34);
        } 
        current_statement__ = 134;
        lp_accum__.add(paramvar);
```
And if there's an extra `dead_code_elimination` right before the first `copy_propagation` then the optimizer generates
```cpp
        double paramvar;
        
        {
          {
            sym112__ = (theta * 34);
          }
        }
        current_statement__ = 85;
        lp_accum__.add(sym112__);
```